### ISSUE TITLE
Body indentation errors undetected when <script> tag exists in <head>

### ISSUE THREAD

**Issue by jbree** [yeonjuan/html-eslint#44]
Detection of indentation errors inside `<body>` seem dependent on contents of `<head>`. Indentation errors should appear for the `<div ...>` and `</div>` lines, but they do not.

Minimal sample configuration:

```js
module.exports = {
  plugins: ["@html-eslint"],
  parser: "@html-eslint/parser",
  rules: {
    "@html-eslint/indent": ["error", 2],
  }
};
```

Test file, div indentation errors unreported:
```html
<!DOCTYPE html>
<html lang='en-US'>
  <head>
    <title>Test Title</title>
    <script src='http://somescript.js'></script>
  </head>
  <body>
      <div id='hello'>
      </div>
  </body>
</html>
```

If the `<script ...></script>` line is removed or commented, indentation errors are properly detected.

**Comment by yeonjuan** [comment]
Hi @jbree There was a bug. It fixed and release in v0.8.0.  🎉
Thanks for reporting it :) 

- [playground](https://yeonjuan.github.io/html-eslint/playground#eyJjb2RlIjoiPCFET0NUWVBFIGh0bWw+XG48aHRtbCBsYW5nPSdlbi1VUyc+XG4gICAgPGhlYWQ+XG4gICAgICAgIDx0aXRsZT5UZXN0IFRpdGxlPC90aXRsZT5cbiAgICAgICAgPHNjcmlwdCBzcmM9J2h0dHA6Ly9zb21lc2NyaXB0LmpzJz48L3NjcmlwdD5cbiAgICA8L2hlYWQ+XG4gICAgPGJvZHk+XG4gICAgICA8ZGl2IGlkPSdoZWxsbyc+XG4gICAgICA8L2Rpdj5cbiAgICA8L2JvZHk+XG48L2h0bWw+IiwicnVsZXMiOnsiQGh0bWwtZXNsaW50L2luZGVudCI6ImVycm9yIiwiQGh0bWwtZXNsaW50L3JlcXVpcmUtbGktY29udGFpbmVyIjoiZXJyb3IifX0=)

```html
<!DOCTYPE html>
<html lang='en-US'>
    <head>
        <title>Test Title</title>
        <script src='http://somescript.js'></script>
    </head>
    <body>
      <div id='hello'> <!-- error -->
      </div>  <!-- error -->
    </body>
</html>
```
